src/api.js

```diff
@@ -0,0 +1,13 @@
+const url = 'https://mate-api.herokuapp.com';
+
+export function getAll() {
+  return fetch(`${url}/todos`)
+    .then(response => response.json())
+    .then(result => result.data);
+}
+
+export function getUsers(userId) {
```

Comment:
```suggestion
export function getUser(userId) {
```
ты получаешь одного пользователя по его id